src/app/components/EmbedConsentBanner/ConsentBanner.styles.tsx

```diff
@@ -25,16 +26,16 @@ export default {
       a: {
         color: 'inherit',
         textDecoration: 'none',
-        borderBottom: `1px solid ${palette.GREY_10}`,
+        borderBottom: `0.0625rem solid ${palette.GREY_10}`,
```

Comment:
you could use [pixelsToRem](https://github.com/bbc/simorgh/blob/latest/src/app/utilities/pixelsToRem/index.ts) so it's more obvious it's a 1px border

Reply:
Ah nice, much better 👍🏼